haproxy/README.md

```diff
@@ -148,3 +149,19 @@ Set `haproxy.config` to `redisk8s` to use the Redis configuration with DNS servi
 | `haproxy.redisk8s.dnsservicename` | The DNS Record for service discovery | `redis-access-headless`
 | `haproxy.redisk8s.nodeCount` | Max number of nodes in the backend | `3`
 | `haproxy.redisk8s.port` | Port of the Galera node | `6379`
+| `haproxy.redisk8s.metrics.enabled`| If the metric endpoint of the Redis backends should be expoded in haproxy | `false`
+
+
+## Galera and Redis metrics
+
+By setting the `haproxy.galerak8s.metrics.enabled` or `haproxy.redisk8s.metrics.enabled` parameters, you can expose the metrics of the backend pods.
+
+For Galera the endpoints are:
+* `<haproxy_service_ip>/metrics/mariadb-0/metrics`
+* `<haproxy_service_ip>/metrics/mariadb-1/metrics`
+* `<haproxy_service_ip>/metrics/mariadb-2/metrics`
+
+For Redis the endpoints are:
+* `<haproxy_service_ip>/metrics/redis-0/metrics`
+* `<haproxy_service_ip>/metrics/redis-1/metrics`
+* `<haproxy_service_ip>/metrics/redis-2/metrics`
```

Comment:
Probably worth mentioning that the metrics frontend listens on 9090.

Reply:
Good point. Adjusted the readme as well as the commit message